perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -376,9 +376,19 @@ sub inspect : Private {
     $c->stash->{report_meta} = { map { 'x' . $_->{name} => $_ } @{ $c->stash->{problem}->get_extra_fields() } };
 
     if ($c->cobrand->can('council_area_id')) {
-        my $priorities_by_category = FixMyStreet::App->model('DB::ResponsePriority')->by_categories($c->cobrand->council_area_id, @{$c->stash->{contacts}});
+        my $priorities_by_category = FixMyStreet::App->model('DB::ResponsePriority')->by_categories(
+            $c->stash->{contacts},
+            area_id => $c->cobrand->council_area_id
+        );
         $c->stash->{priorities_by_category} = $priorities_by_category;
-        my $templates_by_category = FixMyStreet::App->model('DB::ResponseTemplate')->by_categories($c->cobrand->council_area_id, @{$c->stash->{contacts}});
+        my $templates_by_category = FixMyStreet::App->model('DB::ResponseTemplate')->by_categories(
+            $c->stash->{contacts},
+            area_id => $c->cobrand->council_area_id,
+            # The cobrand may wish to provide its own list of bodies independent
+            # of council_area_id - e.g. TfL covers many areas but we only want
+            # to show response templates directly owned by the TfL body.
+            body_ids => $c->cobrand->call_hook('response_template_body_ids')
```

Comment:
There is a gotcha here, in that call_hook returns an empty list in a list context like here, so if there were any arguments after this one (e.g. it was `body_ids => $c->cobrand->call_hook(...), foo => 123`), then body_ids would be set to "foo" and "123" to undef. Adding `|| undef` to the end of the line would prevent.

Reply:
Good catch, thanks.